### PR TITLE
fix: output message with litteral port. fix #90

### LINKED ISSUE #90: The $PORT in the log message is a literal
Hi folks!

When deploying this service on Cloud Run, the `$PORT` is printed as a literal in the log:

```
2024/04/17 14:13:36 Hello from Cloud Run! The container started successfully and is listening for HTTP requests on $PORT
```

Shall this be formatted?

https://github.com/GoogleCloudPlatform/cloud-run-hello/blob/7e45d10641efba44275ceed0c0aff2cc60a2866c/hello.go#L188

Thanks for your support!

### PR DESCRIPTION
**Pull request by JulienBreux**
Fix issue #90

### CODE REVIEW

**Comment by JulienBreux** [comment]
cc @steren @grayside

File: hello.go [hunk 1]
```diff
@@ -184,7 +184,8 @@ func main() {
 	if port == "" {
 		port = "8080"
 	}
+	address := fmt.Sprintf(":%s", port)
 
-	log.Print("Hello from Cloud Run! The container started successfully and is listening for HTTP requests on $PORT")
-	log.Fatal(http.ListenAndServe(fmt.Sprintf(":%s", port), nil))
+	log.Printf("Hello from Cloud Run! The container started successfully and is listening for HTTP requests on %s", port)
```
Comment (grayside) [inline hello.go:189]: suggestion: To keep this approachable to folks newer to HTTP server creation, let's be really clear what the number means.

```suggestion
	log.Printf("Hello from Cloud Run! The container started successfully and is listening for HTTP requests on port %s", port)
```

**Review by grayside — COMMENTED** [review]
Holding approve for the minor improvement.

**Comment by Deleplace** [comment]
I confirm the bug is real, and the fix is good (tested, verified)

**Review by grayside — APPROVED** [review]

**Pull request merged**